src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -76,4 +86,22 @@ object Lorre extends App with LazyLogging {
   }
 
 
+  def processTezosAverageFees(): Try[Unit] = {
```

Comment:
This file should only contain the minimal logic for running Lorre. Please move this function to `TezosDatabaseOperations`.